### PR TITLE
INFRA-331: Add debian packages files 

### PR DESCRIPTION
**Pull request by chinzou**
Add the following packages structures and essential files: 
- usb-autorunner-dev
- usb-autorunner
- usb-autorunner-profiles

### CODE REVIEW

File: usb-autorunner-dev/usb-autorunner [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        ;;
+    esac
+}
+
+config () {
+    case ${1} in
+        -c)
+        gen_cert
+        ;;
+        -t)
```
Comment (chinzou) [inline usb-autorunner-dev/usb-autorunner]: To be deleted

File: usb-autorunner-dev/usb-autorunner [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        echo "Not implemented yet"
+    esac
+}
+
+help() {
+    echo "This is HELP :)"
```
Comment (chinzou) [inline usb-autorunner-dev/usb-autorunner]: Add command line help

File: usb-autorunner-profiles/debian/control [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Priority: optional
+Maintainer: Mekom Solutions <support@mekomsolutions.com>
+Build-Depends: debhelper (>= 10)
+Standards-Version: 4.1.2
+Homepage: https://github.com/mekomsolutions/usb-autorunner
+
+Package: usb-autorunner-profiles
+Architecture: all
+Depends: ${misc:Depends}, usb-autorunner-dev
+Description: <insert up to 60 chars description>
```
Comment (chinzou) [inline usb-autorunner-profiles/debian/control]: Add description

File: usb-autorunner-profiles/debian/copyright [hunk 4]
```diff
@@ -0,0 +1,27 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```
Comment (chinzou) [inline usb-autorunner-profiles/debian/copyright:1]: @rbuisson What license should we use?

File: usb-autorunner-dev/usb-autorunner [hunk 5]
```diff
@@ -26,13 +30,23 @@ config () {
         -c)
         gen_cert
         ;;
-        -t)
-        echo "Not implemented yet"
+        *)
+        echo "option not valid"
```
Comment (rbuisson) [inline usb-autorunner-dev/usb-autorunner]: "Option not valid"

File: usb-autorunner-dev/usb-autorunner [hunk 6]
```diff
@@ -43,11 +57,11 @@ case ${option} in
...
         ;;
-    help) 
+    help)
         help
-        ;; 
-    *)  
-        echo "option '${1}' not found" 
+        ;;
+    *)
+        echo "command '${1}' not found"
```
Comment (rbuisson) [inline usb-autorunner-dev/usb-autorunner]: "Command" ?

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: usb-autorunner-dev/usb-autorunner (hunk 6 above)
Comment (chinzou) [reply usb-autorunner-dev/usb-autorunner]: Commands such as: `generate`, `config` and `help`
While Options are arguments for the commands (eg. `-p`, `-c`...)

Comment (rbuisson) [reply usb-autorunner-dev/usb-autorunner]: This is just about the upper case "C".

**Pull request merged**